text/0000-icons.md

```diff
@@ -0,0 +1,137 @@
+- Start Date: 2021-03-14
+- RFC PR:
+- TVA Issue:
+
+# Summary
+
+Bundle icons as inline SVG into a simple, organized structure, ready for import and use in an application.
```

Comment:
There isn't enough information here.

1. What is the purpose of the icons package?
2. What tools does it use?
3. What problem are we solving by creating this package?